tests/tests/test_stdcm.py

```diff
@@ -266,7 +266,7 @@ def test_max_running_time(small_scenario: Scenario, fast_rolling_stock: int):
     r = requests.post(url, json=payload)
     response = r.json()
     assert r.status_code == 200
-    assert response == {"status": "path_not_found"}
+    assert response["status"] == "conflicts"
```

Comment:
Should we check the list?

Reply:
Thanks, I'm not sure about the details (times for instance), I'll let @eckter check those.